test/integration/common/StudioActions.kt

```diff
@@ -47,51 +47,12 @@ import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 import kotlin.test.fail
 import kotlinx.coroutines.delay
-import kotlinx.coroutines.runBlocking
-
-object Utils {
-    val SAVE_ICON_STRING = Icon.Code.FLOPPY_DISK.unicode
-    val PLUS_ICON_STRING = Icon.Code.PLUS.unicode
-    val PLAY_ICON_STRING = Icon.Code.PLAY.unicode
-    val CHECK_ICON_STRING = Icon.Code.CHECK.unicode
-    val ROLLBACK_ICON_STRING = Icon.Code.ROTATE_LEFT.unicode
-    val CHEVRON_UP_ICON_STRING = Icon.Code.CHEVRON_UP.unicode
-    val DOUBLE_CHEVRON_DOWN_ICON_STRING = Icon.Code.CHEVRONS_DOWN.unicode
-    val DOUBLE_CHEVRON_UP_ICON_STRING = Icon.Code.CHEVRONS_UP.unicode
-
-    val SAMPLE_DATA_PATH = File("test/data/sample_file_structure").absolutePath
-    val TQL_DATA_PATH = File("test/data").absolutePath
-
-    const val QUERY_FILE_NAME = "query_string.tql"
-    const val DATA_FILE_NAME = "data_string.tql"
-    const val SCHEMA_FILE_NAME = "schema_string.tql"
-
-    const val FAIL_CONNECT_TYPEDB = "Failed to connect to TypeDB."
-    const val FAIL_CREATE_DATABASE = "Failed to create the database."
-    const val FAIL_DATA_WRITE = "Failed to write the data."
-    const val FAIL_SCHEMA_WRITE = "Failed to write the schema."
-
-    fun runComposeRule(compose: ComposeContentTestRule, rule: suspend ComposeContentTestRule.() -> Unit) {
-        runBlocking { compose.rule() }
-    }
 
-    fun studioTest(compose: ComposeContentTestRule, funcBody: suspend () -> Unit) {
-        runComposeRule(compose) {
-            setContent { Studio.MainWindowContent(WindowContext.Test(1000, 1000, 0, 0)) }
-            funcBody()
-        }
-    }
-
-    fun studioTestWithRunner(compose: ComposeContentTestRule, funcBody: suspend (String) -> Unit) {
-        val typeDB = TypeDBCoreRunner()
-        typeDB.start()
-        val address = typeDB.address()
-        runComposeRule(compose) {
-            setContent { Studio.MainWindowContent(WindowContext.Test(1000, 1000, 0, 0)) }
-            funcBody(address)
-        }
-        typeDB.stop()
-    }
+object StudioActions {
+    private const val FAIL_CONNECT_TYPEDB = "Failed to connect to TypeDB."
+    private const val FAIL_CREATE_DATABASE = "Failed to create the database."
+    private const val FAIL_DATA_WRITE = "Failed to write the data."
+    private const val FAIL_SCHEMA_WRITE = "Failed to write the schema."
```

Comment:
I quite like the name `StudioActions` as it represents the actions we can take in Studio when running the integration tests. However, these constants feel a bit out of place.

I think we should extract them into, maybe, a sub-object called `Errors`?

Reply:
Done.